src/content/troubleshooting/faq/unrecognized-key-x-in-chromatic-config.mdx

```diff
@@ -0,0 +1,10 @@
+---
+layout: "../../../layouts/FAQLayout.astro"
+sidebar: { hide: true }
+title: I hit the error Unrecognized key {X} in chromatic.config.json
+section: "uiTestsAndReview"
```

Comment:
@winkerVSbecks I was unsure about this, I know its in the Doc troubleshooting category but I was unsure how to designate it. Let me know and I'll make that change right away

Reply:
sorry we should have been clearer @irashid1 . @cgbl-90 created the category 'doc troubleshooting' where we would find places for it in the docs rather than FAQ .. but I believe this should fall under 'continuous integration' .. @winkerVSbecks can confirm